filebeat/harvester/reader/log.go

```diff
@@ -28,13 +28,13 @@ type logFileReader struct {
 }
 
 type LogFileReaderConfig struct {
-	CloseOlder         time.Duration
-	BackoffDuration    time.Duration
-	MaxBackoffDuration time.Duration
-	BackoffFactor      int
-	CloseRenamed       bool
-	CloseRemoved       bool
-	CloseEOF           bool
+	Backoff       time.Duration
+	MaxBackoff    time.Duration
+	BackoffFactor int
```

Comment:
The naming change is to make reader configs consitent with the harvester config names.

+1 on the suggested struct, but then we should have it in libbeat.